chapter_4/exercise_4_04/stack.c

```diff
@@ -158,10 +158,16 @@ void swap(void)
 
 void clear(void)
 {
-  do
+/*  do
   {
     stack[sp] = 0.0;
   } while (sp--);
+*/
+  // Replacing the above commented code with this one ensures messages of "stack empty" aren't displayed an extra time after c command is called in the program
```

Comment:
@sadeem-albir Remove the commented code before merging.

Reply:
Commented code has been removed.